cmd/flakechecker/bq_client.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+	"time"
+)
+
+const projectID = "acs-san-stackroxci"
+const queryTimeout = 1 * time.Minute
+const queryStrGetFailureRatio = `
+SELECT
+    JobName,
+    FilteredName,
+    Classname,
+    TotalAll,
+    FailRatio
+FROM
+` + "`acs-san-stackroxci.ci_metrics.stackrox_tests_recent_flaky_tests`" + `
```

Comment:
```suggestion
` + projectID + "`.ci_metrics.stackrox_tests_recent_flaky_tests`" + `
```

Reply:
I'll leave this as it is. Value `acs-san-stackroxci` has two different purposes:
- constant is used as information for a project where authorization will be done
- in SQL, it represents a project where data is stored

They can be different, and I would prefer not to couple them.